### ISSUE TITLE
Remote run cannot resolve dependencies

### ISSUE THREAD

**Issue by tbelhalfaoui** [guildai/guildai#101]
I cannot run a remote operation that requires a dependency, even if this dependency is present and correctly recognized on the remote.

### Minimal example to reproduce
guild.yml
```
upstream:
  main: upstream
downstream:
  main: downstream
  requires:
    - operation: upstream
```

```
$ guild run upstream --remote dl

$ guild runs --remote dl
[1:d173aee2]   gpkg.anonymous-e25a79aa/upstream              2019-12-18 13:36:16  completed

$ guild run --remote dl downstream upstream=d173aee2
WARNING: cannot find a suitable run for required resource 'upstream'
You are about to run downstream on dl
  upstream: d173aee2
...
Starting downstream on dl as 0bf56444e00f41218524d3df8efa1dd7
WARNING: cannot find a suitable run for required resource 'upstream'
Run 0bf56444e00f41218524d3df8efa1dd7 stopped with a status of 'error'
```

**Comment by gar1t** [comment]
Thank you for the report - and I'm sorry you're facing this issue!

I have recreated this on 0.7.0.rc2 - here's the issue resolution project:

https://github.com/guildai/issue-resolution/tree/master/101-remote-run-cannot-resolve-dependencies

This is a high priority bug that will block 0.7 from shipping - so stay tuned for a fix!

**Comment by tbelhalfaoui** [comment]
Hi @gar1t ! Any update on this? :pray:

**Comment by gar1t** [comment]
This is fixed in master and will be available in rc10. Thanks for your patience!

**Comment by gar1t** [comment]
The fix is now generally available in 0.7.0. If this is still an issue, please re-open and we'll sort it out. Thanks again for the report.
